python_tests/pytest_input_generators.py

```diff
@@ -19,6 +26,76 @@
 MAX_VECTOR_SIZE = 8
 
 
+def _extremal_values(dtype: torch.dtype):
+    _float_vals = (float("inf"), float("-inf"), float("nan"))
+    _complex_vals = tuple(
+        complex(*x) for x in itertools.product(_float_vals, _float_vals)
+    )
+
+    if dtype in (torch.float16, torch.bfloat16, torch.float32, torch.float64):
+        return _float_vals
+    elif dtype in (torch.complex64, torch.complex128):
+        return _complex_vals
+    else:
+        raise ValueError(f"Unsupported dtype --- {dtype}")
+
+
+def _large_values(dtype: torch.dtype):
+    _int_vals = (-1113, 1113, -10701, 10701)
```

Comment:
nitpick, these are not very large :laughing: 

Should we look at max int 32 and things like that? or maybe those should have been in `_extremal_values`?

Reply:
Covering int max values in `_extremal_values` is a good idea.

The int values are relatively small because we also test `int16` whose range is [-32768, +32767].